index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    
+    <div class="container">
+        <div class="first">
+            <img class="x" src="./1.jpg" onclick="openImage('1')" >
```

Comment:
Can you add images from your script file?
So there will be an array of images and then the function will render it. @ShivanshSinghnnovative